requirements.txt

```diff
@@ -17,8 +17,8 @@ coverage==4.2
 ddt==1.1.1
 debtcollector==1.9.0
 decorator==4.0.10
-Django==1.10.7
--e git+https://github.com/open-craft/django-angular.git@5f3e54f909b925eecb59dc4dd73b6e9fe27e9558#egg=django_angular
+Django==1.11.20
+-e git+https://github.com/jrief/django-angular.git@3a4eb9e716b0f03e4abe5ec60d6978a52da100f8#egg=django_angular
```

Comment:
No clue to be honest. I just updated the commit reference. @clemente can we remove this?

Reply:
It was explicitly added [„to prevent installing stable versions“](https://github.com/open-craft/opencraft/commit/11bb21e6cb4dc64600835385e503a521786147e3) though I'm not sure what would happen without the `-e` (I though it would still point to a specific commit)
It was already there and working, so better don't change it in this PR (if we change it, we need to change it for the other dependency that uses it).